src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -29,6 +29,41 @@ import { getLocationMarker } from '../../settings';
 
 const { INPUT_KEYS } = consts;
 
+export const locationServiceEnabledAlert = ({
+  currentPosition,
+  locationServiceEnabled,
+  navigation
+}: {
+  currentPosition?: Location.LocationObject;
+  locationServiceEnabled?: Boolean;
+  navigation: StackNavigationProp<any>;
+}) => {
+  if (!locationServiceEnabled || !currentPosition) {
+    Alert.alert(
+      texts.settingsTitles.locationService,
+      texts.settingsContents.locationService.onSystemPermissionMissing,
+      [
+        {
+          text: texts.sue.report.alerts.ok
```

Comment:
I guess this should not be "OK" but something like "Abbrechen", because the action leads to nothing